RileyLink/RileyLinkBLEDevice.h

```diff
@@ -11,9 +11,9 @@
 
 
 typedef NS_ENUM(NSUInteger, RileyLinkState) {
-  RILEY_LINK_STATE_CONNECTING,
-  RILEY_LINK_STATE_CONNECTED,
-  RILEY_LINK_STATE_DISCONNECTED
+  RLStateConnecting,
+  RLStateConnected,
+  RLStateDisconnected
 };
```

Comment:
So the naming conventions state that each type should begin with the enum type name, e.g. `RileyLinkStateConnecting`. This automatically bridges to swift enums as `case Connecting`